app/Controllers/Admin/SettingController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using backend.Dto.Requests;
+using backend.Services.Admin;
+using backend.Helpers;
+
+namespace backend.Controllers.Admin
+{
+    [Route("admin/settings")]
+    [ApiController]
+    public class SettingController : ControllerBase
+    {
+        private readonly SettingService settingService;
+
+        public SettingController(SettingService settingService)
+        {
+            this.settingService = settingService;
+        }
+
+
+        // GET admin/settings
+        [HttpGet()]
+        [Authorize(AccountTypeEnum.Admin)]
+        public async Task<IActionResult> GetSettings()
+        {
+            return Ok(await this.settingService.Get());
+        }
+
+        // PUT admin/settings
+        [HttpPut()]
```

Comment:
We can probably change HttpPut() to HttpPut (similar to VaccinationSlotController).